docs/basics/part-7-using-remote-runners.md

```diff
@@ -0,0 +1,88 @@
+
+## Remote runners
+
+Earthly is able to use remote runners for performing builds on remote machines. When Earthly uses a remote runner, the inputs of the build are picked up from the local environment, then the execution takes place remotely, including any pushes (`RUN --push` commands, and `SAVE IMAGE --push` commands), but any local outputs are sent back to the local environment. All this takes place while your local Earthly process still provides the logs of the build in real time locally.
+
+Remote runners are especially useful in a few specific circumstances:
+
+* You want to reuse cache between CI runs to dramatically speed up builds (more on this in part 8).
+* You want to share compute and cache with colleagues and/or with the CI.
+* You have a build that requires a lot of resources, and you want to run it on a machine with more resources than your local machine.
+* You have a build that requires to run on a specific CPU architecture natively.
+* You have a slow internet connection.
+
+There are two types of remote runners:
+
+* Remote Buildkit (self-hosted)
+* Earthly Satellites
+
+### Using a remote Buildkit
+
+A common way to use remote runners is to deploy your own instance of Buildkit and have Earthly connect to it. The [remote Buildkit page](../ci-integration/remote-buildkit.md) has more information on how to set this up.
+
+Once the remote Buildkit is up and running, you may use it from Earthly by setting the configuration option `buildkit_host` to the address of the remote Buildkit. For example, if the remote Buildkit is running on `earthly-remote-buildkit.example.com`, you may set the configuration option
+
+```bash
+earthly config set global.buildkit_host earthly-remote-buildkit.example.com
+```
+
+And then run Earthly builds as usual.
+
+```bash
+earthly +my-target
+```
+
+### Using an Earthly Satellite
+
+Another way to use remote runners is to use Earthly Satellites. Earthly Satellites are remote runners managed by the Earthly team. They are a paid feature as part of the [Earthly Satellites or Earthly CI plans](https://earthly.dev/pricing).
+
+To get started, first you need create an Earthly Cloud account.
+
+```bash
+earthly account register --email <your-email>
+```
+
+Then, follow instructions in the email received. Then, create an organization.
+
+```bash
+earthly org create my-org
+```
+
+Then, subscribe to a paid plan. The subscription has a 14-day trial -- your credit card is not charged if you cancel before then.
+
+[**Click here to start your subscription**](https://buy.stripe.com/8wM9Es4BT4Vvb4YbIJ)
+
+Then, you can create a satellite.
+
+```bash
+earthly sat launch my-satellite
+```
+
+You can then select the satellite via
```

Comment:
I believe the satellite that is last launched is selected automatically.
I understand it's good to introduce users to the select subcommand, but I think it might be worth mentioning.

Reply:
Done